### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.7][IntelliJ][Wipro]: Keyboard focus is not visible on "Help" button in  push image window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1443]
**Repro Steps:**

1. Create a new dynamic web project.
2. Navigate to and select Project ->Azure->Push Image.
3. Push image dialog will open.
4. Tab to Help (?) button and hit Space bar and observe keyboard focus .

**Expected Result:**
Keyboard focus is should visible on "Help" button in push image  window.

**Actual Result:**
Keyboard focus is not visible on "Help" button in push image window.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![no focus visible on help button in push image](https://user-images.githubusercontent.com/22956130/35729098-7f860b5e-07c2-11e8-863b-a1659258b1b0.png)

**Comment by Eskibear** [comment]
duplicate with https://github.com/Microsoft/azure-tools-for-java/issues/1456

**Comment by WiproPSAccTest** [comment]
Verified in 2017.3.5 Community.
Issue still exists
![image](https://user-images.githubusercontent.com/22956130/39819232-91559256-53c0-11e8-91b0-af3d4f369064.png)

**Comment by jdneo** [comment]
* **Importance of experience**: minimal. Very low usage for the help action.
* **Quality of experience**: not impacted. User can still push his image into ACR. If he wants to know the meaning of what's each field. He can check the docs for ACR.
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749896
